main.lua

```diff
@@ -10,6 +10,7 @@ import "checkBoxButtons"
 local gfx <const> = playdate.graphics
 local menu = playdate.getSystemMenu()
 
+local congratulationsTimer = nil
```

Comment:
don't really like global vars, but if exit to menu w/o waiting for the timer finish, then it will still be triggered once 10 seconds pass. We do remove the sprites later down the load, but the timer will still exist and will trigger the logic behind 'congratulations sprite'